src/tenant/views.py

```diff
@@ -114,3 +133,30 @@ def post(self, request, *args, **kwargs):
     def get_success_url(self):
         """ Redirect to the newly created tenant."""
         return self.object.get_root_url()
+
+
+@receiver(email_confirmed, sender=EmailConfirmationHMAC)
+def email_confirmed_handler(email_address, **kwargs):
+    # Once the owner user has verified the email for the first time,
+    # send an email with instructions for how to log in with the username and password.
+    user = email_address.user
+
+    # just verified email for a first time and never been logged into app before
+    if user.last_login is not None:
+        return
+    if user.is_staff is not True:
```

Comment:
only the `siteconfig.owner`, not all staff.

Reply:
done